libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardItem/DashboardItemHeadlineContainer.tsx

```diff
@@ -1,10 +1,33 @@
 // (C) 2020 GoodData Corporation
 import React from "react";
 
-export const DashboardItemHeadlineContainer: React.FC = ({ children }) => {
+interface IDashboardItemHeadlineContainerProps {
+    children: React.ReactNode;
+    clientHeight?: number;
+}
+
+const SMALL_CUSTOM_HEIGHT = 120;
+const SMALL_HEIGHT = 38;
+const SMALL_LINE_HEIGHT = 36;
+
+export const DashboardItemHeadlineContainer: React.FC<IDashboardItemHeadlineContainerProps> = ({
+    children,
+    clientHeight,
+}) => {
+    const isSmallCustomHeight = () => clientHeight < SMALL_CUSTOM_HEIGHT;
+
     return (
-        <div className="item-headline-outer">
-            <div className="item-headline">{children}</div>
+        <div
+            className={"item-headline-outer"}
+            style={
+                isSmallCustomHeight()
+                    ? { height: `${SMALL_HEIGHT}px`, lineHeight: `${SMALL_LINE_HEIGHT}px` }
```

Comment:
check this inline style and comment in the item-headline-outer css class: "height: ($item-headline-lineHeight + $item-headline-padding-vertical) * 2 + 2; // relate SD-956: please update HEADLINE_OUTER_HEIGHT in widgetHeightUtil.ts when height is changed"

Reply:
Is used for GeoPushPin, afaik it wont be affected for this small heights.